core/src/main/java/org/apache/iceberg/TableMetadata.java

```diff
@@ -431,7 +446,10 @@ public TableMetadata addStagedSnapshot(Snapshot snapshot) {
         .build();
 
     return new TableMetadata(null, formatVersion, uuid, location,
-        snapshot.sequenceNumber(), snapshot.timestampMillis(), lastColumnId, schema, defaultSpecId, specs, properties,
+        snapshot.sequenceNumber(),
+        //TODO: should this be System.currentTimeMillis()
```

Comment:
We use the one from the new snapshot so that the timestamps all match. Do you think we should change that?

Reply:
I think it is ok.

Reply:
If it's okay, can you remove this change?